### ISSUE TITLE
Feature request: Add option to overwrite original files.

### ISSUE THREAD

**Issue by vovan69** [plegall/Piwigo-write_metadata#7]
By default exiftool saves backup of the original file. It would be nice to have an option to turn this off (exiftool option -overwrite_original).

**Comment by erkannt** [comment]
Not a php-dev, so no idea how to make this an option. I just edited the main.inc.php in the plugin folder:

`main.inc.php:  $command = isset($conf['exiftool_path']) ? $conf['exiftool_path'] : 'exiftool -overwrite_original';`

Works fine for me.

**Comment by pjgaus** [comment]
I agree that this should be an option.  I end up going back and deleting the original files.
